### ISSUE TITLE
Function `as_next_scheduled_action` Incorrectly returns false when arguments have slashes in them.

### ISSUE THREAD

**Issue by scotteuser** [Prospress/action-scheduler#358]
When args for a scheduled action have a slash in them, `as_next_scheduled_action` incorrectly returns false even when there is a pending action.
```php
function test_action_scheduling() {

	// Create a single action WITH a slash in it.
	$hook_name = 'test_hook_name_with_slash';
	$timestamp = time();
	$args      = [
		'offset'          => 'something/something-else',
		'previous_offset' => 'another thing',
	];
	$group     = 'mygroup';
	as_schedule_single_action( $timestamp, $hook_name, $args, $group );
	var_dump( as_next_scheduled_action( $hook_name, $args, $group ) );
	// RESULT: bool(false)


	// Create a single action WITHOUT a slash in it.
	$hook_name = 'test_hook_name_without_slash';
	$timestamp = time();
	$args      = [
		'offset'          => 'something something-else',
		'previous_offset' => 'another thing',
	];
	$group     = 'mygroup';
	as_schedule_single_action( $timestamp, $hook_name, $args, $group );
	var_dump( as_next_scheduled_action( $hook_name, $args, $group ) );
	// RESULT EG: int(1568301286)
	die();
}

add_action( 'init', 'test_action_scheduling' );
```

This is because action scheduler is comparing:
```
{"offset":"something/something-else","previous_offset":"another thing"}
```
with 
```
{"offset":"something\/something-else","previous_offset":"another thing"}
```
resulting in a non-match.

**Comment by scotteuser** [comment]
@rrennick I have tested it now and it works correctly on the 3.0.0 beta branch. Is this something that is possible to backport or is 3.0.0 close to release?

**Comment by rrennick** [comment]
@scotteuser Thanks for the follow up. We are not planning on back porting any of the fixes that are currently in 3.0.0. We are looking to release later this month or early October. I'll close this as resolved.
